src/pages/About/index.js

```diff
@@ -1,5 +1,36 @@
-import { Wrapper } from "./style";
-
+import { Card, CardsContainer, HighlightText, Wrapper } from "./style";
+import ComunnityImg from '../../assets/community.svg';
+import EmpowermentImg from '../../assets/empowerment.svg';
+import LightBoxImg from '../../assets/light-box.svg';
 export const About = () => {
-  return <Wrapper>About</Wrapper>;
+  return (
+    <Wrapper>
+          <h2>SOBRE NÓS</h2>
+          <p>Somos uma comunidade voltada a incentivar a cultura open source no curso</p>
+          <p>de Ciência da Computação da UFCG</p>
+          <CardsContainer class="about-us-container">
+            <Card class="about-us-card">
```

Comment:
Como a gente ta usando styled components a gente não precisa geralmente usar classes :)

```suggestion
            <Card>
```

Acho inclusive que tu pode criar um componente nessa mesma pasta com o nome `Card` que serve pra encapsular essa lógica do cardizinho interno desse componente, aí fica mais organizado mesmo. Aí esse componente ficaria em `src/components/AboutSection/Card/Index.js`